packages/git/.gitconfig.aliases

```diff
@@ -0,0 +1,24 @@
+[alias]
+  #
+  l = ! git log --oneline --color
+  lg = ! git log --oneline --graph --decorate --color | emojify | less -RFe
```

Comment:
Not really useful, trim it to `lg = ! git log --oneline --graph --decorate --color`